css/style.css

```diff
@@ -5,12 +5,21 @@
 
 .tittle-brand {
     font-family: 'Bebas Neue', cursive;
-    color: #e15361; 
+    color: #666666; 
     width: auto; 
 }
 
 .tittle-brand span {
-    color: #666666;
+    color: #e15361;
+}
+
+.navbar-brand {
+    margin: 0.5rem;
+}
+
+.navbar-brand span {
+    color:#e15361;
+    font-family: 'Bebas Neue', cursive;
 }
 
 .custom-focus:focus {
```

Comment:
Unnecessary spacing within this CSS declaration, remove all unnecessary white-spacing.

Reply:
fixed all the empty space